doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-configuring-EAP-security-management-provider-proc.adoc

```diff
@@ -0,0 +1,67 @@
+[id='business-central-configuring-EAP-security-management-provider-proc']
+
+= Configuring Wildfly or {EAP} security management provider
+
+.Prerequisites
+
+* Wildfly or {EAP} is installed on the system.
+* If you want to use an existing users or roles property files from the Wildfly or {EAP} instance, you must include the following system properties:
++
+.Wildfly or {EAP} security management provider based on property files
```

Comment:
@RishiRajAnand do we have any use case where user wants to use its own implementation of security management provider? If the business central integrated with RH-SSO, we have special documentation for that.